### DISCUSSION TITLE
Can RadioLib with SX1262 receive message from different frequency?

### DISCUSSION THREAD

**Discussion by chansheunglong** [jgromes/RadioLib#548] [Q&A]
Hi everyone, new to RadioLib. I have been using standard LoRaWAN stuff for a while and noticed that LoRaWAN can receive messages from multiple frequency at the same time (e.g. AS923 receiver can receive messages 923.2Mhz to 924.8Mhz). I am wondering if, more specifically, using RadioLib and SX1262 (or other chips) on both sides, can the receiving end detect and decode messages from a wider frequency range? Or just the signal at the frequency that is set in the setting.

I assume if both sides are set using `radio.begin(915.0, 500.0, 6, 5, 0x34, 20)` , they can only listen to message on 915Mhz, am I correct?

If that is the case, if I would like to build a simple LoRa receiver that can listen to, for example, 2 frequencies (915Mhz and 916Mhz), does that mean I need to use one IC per frequency?

Thanks!

**Comment by jgromes** [comment]
Generally, you never listen on just one frequency - rather in some band. The width of the band you can receive messages from usually depends on the properties of the signal. In the case of LoRa modulation, it is determined by the signal bandwidth: if the frequency is within +-25% of the bandwidth, then the receiver will get the signal (there are some edge cases which have a tighter limit). For example, in the case of a 500 kHz LoRa signal and receiver centered at 915 MHz, the transmitter center frequency can be anywhere between 914.875 MHz and 915.125 MHz for the signal to be received.

If you want to go higher, then you will need some additional logic (a frequency division multiplex). For example, your receiver could switch between multiple channels, listening on each one for a certain period of time. If a preamble is seen, then the receiver can start receiving a packet on that channel. Now the width of the band from which you can receive will depend on the length of the preamble, how fast you can switch the receiver channel, and how many packets are you willing to lose. I'm not familiar with LoRaWAN, but I suspect that this sort of multiplexing could be going on.
